### PR TITLE
made yuricat-AI.

### PR DESCRIPTION
**Pull request by YuriCat**

### CODE REVIEW

File: src/cpu/yuricat/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd "$(dirname "$0")"
+exec ./sample "$@" 2> sample.err
```
Comment (mayah) [inline src/cpu/yuricat/run.sh]: exec ./yuricat "$@" 2> yuricat.err

File: src/cpu/yuricat/main.cc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include "base/base.h"
+#include "core/algorithm/plan.h"
+#include "core/client/ai/ai.h"
+#include "core/core_field.h"
+#include "core/frame_request.h"
+
+class SampleAI : public AI {
```
Comment (mayah) [inline src/cpu/yuricat/main.cc]: Could you rename SampleAI to YuriCatAI or something?

File: src/cpu/yuricat/main.cc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+public:
+    SampleAI(int argc, char* argv[]) : AI(argc, argv, "sample") {}
```
Comment (mayah) [inline src/cpu/yuricat/main.cc]: "sample" -> "yuricat"

File: src/cpu/yuricat/main.cc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                }
+
+                if (score > s) {
+					
+                    score = s;
+                    best = plan.decisions().front();
+                }
+        });
+
+		//std::cerr<<"oh"<<std::endl;
```
Comment (mayah) [inline src/cpu/yuricat/main.cc]: Please try to keep indent.

Comment (YuriCat) [inline src/cpu/yuricat/main.cc]: Xcode settled indent. I should study how to change config of Xcode...
and I want to know what "TAB" is.

Comment (mayah) [inline src/cpu/yuricat/main.cc]: TAB is just '\t'.
http://ja.wikipedia.org/wiki/タブキー

I mean, please use spaces to indent instead of tab.

**Comment by mayah** [comment]
Could you update your code to resolve my comments?
After that, please send PR again.
Actually, I'm OK if TAB character is used in cpu/ code.
However, "run.sh" must be fixed before merging this commit, since it just doesn't work.

File: src/cpu/yuricat/main.cc (hunk 4 above)
Comment (YuriCat) [inline src/cpu/yuricat/main.cc]: I don't use TAB Key in programming. Probably, Xcode automatically generates them. mmm...

**Pull request closed without merging**